network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/rich_text_block.html

```diff
@@ -0,0 +1,11 @@
+{% extends "./base_streamfield_block.html" %}
+
+{% comment %}
+This block is used for rich-text streamfields, such as the 'paragraph'
+and 'content' blocks found on Article, Blog, Campaign, Oppurtunity, and Primary pages. 
+{% endcomment %}
+
+
+{% block block_content %}
+    {{ value }}
```

Comment:
This value will be richtext, right? We'll need to add `{{ value|richtext }}` instead. 

Under the hood, Wagtail stores links to pages, images and documents as non-valid HTML. Such as (and Im paraphrasing here) `<a type="link" page-id="42">Text in here</a>` and `|richtext` internalizes that and looks up the page id from the markup it's given. 

Edited: `{% load wagtailcore_tags %}`